client/src/components/LoginForm/LoginForm.js

```diff
@@ -0,0 +1,88 @@
+import { useState } from 'react'
+import axios from 'axios'
+import styled from 'styled-components/macro'
+import ButtonGreen from '../Buttons/ButtonGreen/ButtonGreen'
+import ErrorCard from '../Messages/ErrorCard/ErrorCard'
+import SuccessCard from '../Messages/SuccessCard/SuccessCard'
+
+const LoginForm = () => {
+  const [username, setUsername] = useState('')
+  const [password, setPassword] = useState('')
+  const [errorMessage, setErrorMessage] = useState('')
+  const [successMessage, setSuccessMessage] = useState('')
```

Comment:
I was wondering if you could use just one state variable for your messages? I think this should also work. 🤔

Reply:
I think both ways are okay. I actually might implement it different in the RegisterAccountForm.
Here the success message is just currently implemented. Later on I want to redirect the user to the profile page.